### PR TITLE
Best Index hackerearth problem with solution

### PR DESCRIPTION
**Pull request by Rune-Coder**

### CODE REVIEW

File: HackerEarth_problems/Best Index/solution.cpp [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+#include <cmath>
+#include <algorithm>
+using namespace std;
+int main()
+{
+	int n, nt;
+	long s = 0, m;
+	cin >> n;
+	int *a = new int[n];
+	long *b = new long[n];
```
Comment (Rune-Coder) [inline HackerEarth_problems/Best Index/solution.cpp:12]: //taken 2 dynamic arrays

**Review by Vishruth-S — CHANGES_REQUESTED** [review]
Could you please recheck your solution.
The output of your code for the testcase
```
6
-3 2 3 -4 3 1
```
is `5`
The expected correct output is `3`

**Comment by Rune-Coder** [comment]
Thank you for mentioning. There was a small error which I now have fixed. Please review the code

**Review by Vishruth-S — APPROVED** [review]
Great job 👍 Thanks for contributing 😄 

PS: do star this repo if you haven't already 🙃

**Pull request merged**